kevinzs/day02/day02.py

```diff
@@ -0,0 +1,20 @@
+def solver(input):
+    res = 0
+    for row in input:
+        min_value = min(int(x) for x in row)
+        max_value = max(int(x) for x in row)
```

Comment:
I would stored `int(x) for x in row` in a variable to just iterate once. Anyway, I don't think it would have a lot of impact.

Reply:
Yes i have done a single `for` to min and max, and I don't use `min` and `max` method from kotlin because it's doing the same as `Kevin`

Reply:
Yeah I was thinking that too, you are doing 4 iterations when only one is needed, but to be honest this is a beautiful solution so chapeau 🎩